### PR TITLE
Update instructions for Trigger Click Events with jQuery

### LINKED ISSUE #8634: Trigger Click Events with jQuery: Missing $ sign will still pass the test.
I Don't know if it is a general issue or only related to this task.
if i write folling code in the exercise:
```
    $("#getMessage").on("click", function() {
      //alert((".message"));
      (".message").html("You clicked get Message");
    });
```
This test will pass, even though i should have put a dollar sign before ("message"). I found this error when I passed the test, but the message didn't change on the mobile screen.
[link](https://www.freecodecamp.com/challenges/trigger-click-events-with-jquery#?solution=%0Afccss%0A%20%20%24%28document%29.ready%28function%28%29%20{%0A%20%20%20%20%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%20%20%20%20%24%28%22%23getMessage%22%29.on%28%22click%22%2C%20function%28%29%20{%0A%20%20%20%20%20%20%2F%2Falert%28%28%22.message%22%29%29%3B%0A%20%20%20%20%20%20%24%28%22.message%22%29.html%28%22You%20clicked%20get%20Message%22%29%3B%0A%20%20%20%20}%29%3B%0A%20%20%20%20%2F%2F%20Only%20change%20code%20above%20this%20line.%0A%20%20}%29%3B%0Afcces%0A%0A%0A%3Cdiv%20class%3D%22container-fluid%22%3E%0A%20%20%3Cdiv%20class%20%3D%20%22row%20text-center%22%3E%0A%20%20%20%20%3Ch2%3ECat%20Photo%20Finder%3C%2Fh2%3E%0A%20%20%3C%2Fdiv%3E%0A%20%20%3Cdiv%20class%20%3D%20%22row%20text-center%22%3E%0A%20%20%20%20%3Cdiv%20class%20%3D%20%22col-xs-12%20well%20message%22%3E%0A%20%20%20%20%20%20The%20message%20will%20go%20here%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%3C%2Fdiv%3E%0A%20%20%3Cdiv%20class%20%3D%20%22row%20text-center%22%3E%0A%20%20%20%20%3Cdiv%20class%20%3D%20%22col-xs-12%22%3E%0A%20%20%20%20%20%20%3Cbutton%20id%20%3D%20%22getMessage%22%20class%20%3D%20%22btn%20btn-primary%22%3E%0A%20%20%20%20%20%20%20%20Get%20Message%0A%20%20%20%20%20%20%3C%2Fbutton%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%3C%2Fdiv%3E%0A%3C%2Fdiv%3E%0A)

#### Browser Information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code

```js
// If relevant, paste all of your challenge code in here

```

### PR DESCRIPTION
**Pull request by nishant-tomer**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue : Closes #8634
#### Description
1. Instructions were provided for two consecutive challenges in a single challenge, while the tests for [first challenge](https://www.freecodecamp.com/challenges/trigger-click-events-with-jquery) were testing only against a subset of its instructions.
   So removed the duplicate instructions.
2. Removed grammatical error in [this line](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json#L13) which wasn't part of reported issue.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
-        "First, let's review what the <code>$(document).ready()</code> function does. This function makes it so all code inside of it only runs  once our page loads.",
-        "Let's make our \"Get Message\" button change the text of the element with the class <code>message</code>.",
-        "Before we can do this, we need to implement a <code>click event</code> inside of our <code>$(document).ready()</code> function by adding this code:",
-        "<blockquote>$(\"#getMessage\").on(\"click\", function(){</br></br>});</blockquote>"
+        "First, let's review what the <code>$(document).ready()</code> function does. This function runs such that all of the code inside of it executes only once our page has finished loading.",
+        "So Let us start by implementing a <code>click event handler</code> inside of our <code>$(document).ready()</code> function by adding this code:",
+        "<blockquote>$(\"#getMessage\").on(\"click\", function(){</br></br>});</blockquote>",
+        "After completing this challenge, proceed to the next challenge where we will make our <code>Get Message</code> button change the text of the element with the class <code>message</code>."
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json]: Grammar and formatting:

```diff
-"After completing this challenge, proceed to the next challenge...
+"After completing this, proceed to the next challenge... 

-<code>Get Message</code>
+\"Get Message\"
```

File: seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
         "In this section, we'll learn how to get data from APIs. APIs - or Application Programming Interfaces - are tools that computers use to communicate with one another.",
         "We'll also learn how to update HTML with the data we get from these APIs using a technology called Ajax.",
-        "First, let's review what the <code>$(document).ready()</code> function does. This function makes it so all code inside of it only runs  once our page loads.",
-        "Let's make our \"Get Message\" button change the text of the element with the class <code>message</code>.",
-        "Before we can do this, we need to implement a <code>click event</code> inside of our <code>$(document).ready()</code> function by adding this code:",
-        "<blockquote>$(\"#getMessage\").on(\"click\", function(){</br></br>});</blockquote>"
+        "First, let's review what the <code>$(document).ready()</code> function does. This function runs such that all of the code inside of it executes only once our page has finished loading.",
+        "So Let us start by implementing a <code>click event handler</code> inside of our <code>$(document).ready()</code> function by adding this code:",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json]: Sounds somewhat odd.

```diff
-"So Let us start...
+"Let's start...
```

**Comment by raisedadead** [comment]
@nishant-tomer can you make some minor changes as suggested. I know this is being a bit nit picky.. :wink: 

When you are done amend you last commit and push.

`git commit --amend`
`git push origin <pr branch name> --force`

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @nishant-tomer , you are on some kind of spree!
